packages/lumx-react/src/components/fieldset/Fieldset.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react';
+
+import classNames from 'classnames';
+
+import { CSS_PREFIX } from '@lumx/core/js/constants';
+import { Size, Theme } from '@lumx/react';
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import { IGenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+
+/**
+ * Defines the props of the component.
+ */
+interface IFieldsetProps extends IGenericProps {
+    legend?: ReactElement | string;
```

Comment:
Could we add some documentation on these props ? Just to know what they are and how they impact on thecomponent